x-pack/plugin/esql/src/main/antlr/EsqlBaseLexer.g4

```diff
@@ -130,6 +130,7 @@ RP : ')';
 TRUE : 'true';
 
 EQ  : '==';
+SEQ  : '=~';
```

Comment:
My first proposal was for `EQ_IGNORE_CASE`, but it was a bit verbose.
We went for `SEQ` only because it's the same we have in EQL

Reply:
I can see how SEQ can be confusing. CEIQ or EQS (Equals String) both work for me.
I prefer string equality since it's more generic and it might go beyond just case insensitive comparisons.

Reply:
Ok, I see a majority for CIEQ, I'll use it